Cargo.toml

```diff
@@ -283,6 +283,7 @@ jsonrpsee = { version = "0.24", features = ["server", "macros", "client", "ws-cl
 leb128 = "0.2.5"
 lru = "0.12"
 mockall = "0.11.4"
+moka = { version = "0.12", default-features = false, features = ["sync", "atomic64"] }
```

Comment:
Move moka to iota-storage/Cargo.toml if it's only used there.

Reply:
Actually `moka` will be used in the future upstream PR like https://github.com/MystenLabs/sui/pull/21164, where `iota-core` will also use it, so I put it here to avoid duplicate work in the future. Does it make sense?